### PR TITLE
OCPBUGS-3923: adjust watch budget for monitoring components

### PR DESCRIPTION
**Pull request by jan--f**

### CODE REVIEW

File: test/extended/apiserver/api_requests.go [hunk 1]
```diff
@@ -203,7 +203,7 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 				"cluster-autoscaler-operator":            72.0,
 				"cluster-baremetal-operator":             86.0,
 				"cluster-image-registry-operator":        160.0,
-				"cluster-monitoring-operator":            60.0,
+				"cluster-monitoring-operator":            65.0,
```
Comment (juzhao) [inline test/extended/apiserver/api_requests.go:206]:
```
"cluster-monitoring-operator":            65.0
```
applies for most 4.13 metal jobs, but sometimes, the watchrequestcount for CMO would exceed 65*2
 https://search.ci.openshift.org/?search=produces+more+watch+requests+than+expected&maxAge=336h&context=1&type=bug%2Bjunit&name=periodic-ci-openshift-release-master-nightly-4.13-e2e-metal.*&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job
```
periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-serial-ovn-ipv6 (all) - 31 runs, 52% failed, 44% of failures match = 23% impact
#1619772037583081472	junit	3 hours ago	

    <[]string | len:1, cap:1>: [
        "Operator \"cluster-monitoring-operator\" produces more watch requests than expected: watchrequestcount=131, upperbound=120, ratio=1.09",
    ]


periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-serial-ovn-dualstack (all) - 31 runs, 39% failed, 17% of failures match = 6% impact
#1619459518469509120	junit	24 hours ago	

    <[]string | len:1, cap:1>: [
        "Operator \"cluster-monitoring-operator\" produces more watch requests than expected: watchrequestcount=135, upperbound=120, ratio=1.13",
    ]


periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-serial-virtualmedia-bond (all) - 31 runs, 29% failed, 22% of failures match = 6% impact
#1619589544519143424	junit	15 hours ago	

    <[]string | len:1, cap:1>: [
        "Operator \"cluster-monitoring-operator\" produces more watch requests than expected: watchrequestcount=136, upperbound=120, ratio=1.13",
    ]
```

Comment (stbenjam) [reply test/extended/apiserver/api_requests.go:206]: I'd suggest 77 for metal (139 * 110% / 2), this value is certainly not enough

```
$ curl https://search.ci.openshift.org/\?search\=.\*Operator.\*cluster-monitoring-operator.\*produces+more+watch+requests+than+expected.\*\&maxAge\=168h\&context\=0\&type\=junit\&name\=metal\&excludeName\=\&maxMatches\=5\&maxBytes\=20971520\&groupBy\=job 2>/dev/null | egrep -o "watchrequestcount=[[:digit:]]+" | cut -d= -f2 | cut -d, -f1 | Rscript -e 'quantile (as.numeric (readLines ("stdin")), probs=c(0.50,0.90,0.95,1.00))'
 50%  90%  95% 100% 
 128  135  136  139
```

**Comment by stbenjam** [comment]
Around the 28th, metal dualstack went to near permafailing because of this, could we fix the values for that one and get this landed? 🙏🏻

File: test/extended/apiserver/api_requests.go (hunk 1 above)
Comment (jan--f) [reply test/extended/apiserver/api_requests.go]: Thanks, I bumped it up to the suggested 77.

**Pull request merged**